test/integrations/validation/Zod.spec.ts

```diff
@@ -0,0 +1,348 @@
+
+import { expect, describe, it } from 'vitest';
+
+import zod from '../../../src/integrations/validation/module';
+
+import { SCHEMAS, ERRORS } from './_fixtures/Zod.fixtures';
+
+describe('Zod', () =>
+{
+    describe('should validate using the String schema', () =>
+    {
+        it('should validate name', () =>
+        {
+            const input = { name: 'John' };
+            zod.validate(input, SCHEMAS.STRING);
```

Comment:
This test doesn't actually test anything... I think we expect it to return nothing at success. Should we add an `expect` for this?